cLIMS/organization/templates/dcicView.html

```diff
@@ -9,40 +9,53 @@
 </span>
  {% endif %}
  <br/> <br/>
- {% if experiments %}
+ {% if expSet %}
 <span class="tableHead">
-	Experiment Details
+  Experiment Details
 </span>
 {% if messages %}
         {% for message in messages %}
             <p class="alert alert-danger">{{message}}</p>
         {% endfor %}
 {% endif %}
+<br/>**If nothing is selected by default it will take all experiments<br/>
```

Comment:
wow ! that a lot of changes @Snehal1894 - that is that DCIC related bug reported by @betulakgol , right ?

could you at least briefly describe what's going on here ?

Reply:
The HTML page is modified to display tables containing exps for each experiment set (expset).
Expset is a dictionary {Expset object: [all exp objects]}.
So for every Experiment Set (key in dictionary) a table is generated , with experiments in list (value of dictionary)